isis/tests/Fixtures.cpp

```diff
@@ -1693,4 +1693,14 @@ namespace Isis {
       testCube = new Cube(testPath, "rw");
     }
   }
+
+void NearMsiCameraCube::Init() {
```

Comment:
Instead of adding a new Init method, override the parent SetUp method and GTest will automatically call this.

Reply:
Fixed.